code/go/0chain.net/blobbercore/handler/handler_test.go

```diff
@@ -248,6 +235,15 @@ func isEndpointRequireSignature(name string) bool {
 	}
 }
 
+func isEndpointUpload(name string) bool {
+	switch name {
+	case "Upload":
+		return true
+	default:
+		return false
+	}
```

Comment:
Could be a simple `return if name == "Upload"` unless there are other possible names can be treated as Upload.